Antidote/AppDelegate.swift

```diff
@@ -10,18 +10,20 @@ import Cocoa
 
 @NSApplicationMain
 class AppDelegate: NSObject, NSApplicationDelegate {
-
-    @IBOutlet weak var window: NSWindow!
-
+    var mainWindowController: MainWindowController!
+    var settingsWindowController: SettingsWindowController!
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         // Insert code here to initialize your application
+        mainWindowController = MainWindowController()
+        mainWindowController.showWindow(nil)
+
+        settingsWindowController = SettingsWindowController()
+        settingsWindowController.showWindow(nil)
```

Comment:
In AppDelegate we create all needed windows.

We don't want to show settings on startup, it is here just as an example.